.github/CHANGELOG.md

```diff
@@ -13,7 +14,7 @@
 12. Bug - Added handoff prediction between Thames and City Director - thanks to @Bucklerz (Reece Buckley) and @kye-taylor (Kye Taylor)
 13. Procedure Change - PC East -> S10 - LIBSO agreement - thanks to @robbo599 (Lee Roberts)
 14. Enhancement - Updated Solent Outboud Agreements - thanks to @AleksMax (Aleks Nieszczerzewski)
-15. Enhancement - Updated EGTF SMR - thanks to @GeekPro101 (Thomas Mills) and @Bucklers (Reece Buckley) 
+15. Enhancement - Updated EGTF SMR - thanks to @GeekPro101 (Thomas Mills) and @Bucklers (Reece Buckley)
```

Comment:
Think you changed this line (removed a space)? But I noticed it says Bucklers and not Bucklerz, if you care... 😄

Reply:
Oops :P Doesn't matter that much